### PR TITLE
Add Send notification lambda

### PR DESCRIPTION
**Pull request by AshwaryaSmridhi**
This PR adds instructions to add a notification lambda. As discussed we want to send notification via SMS and not use slack for the upcoming workshop. In the process I had to add some IAM roles to grant specific permission.
 
If you want to run the working solution, please clone https://github.com/AshwaryaSmridhi/simple-lambda-alert-notification.

### CODE REVIEW

File: workshop-content/03-project-work/03.4-sns-and-sqs-setup/03.4-sns-and-sqs-setup.md [hunk 1]
```diff
@@ -64,6 +82,9 @@ resources: # CloudFormation template syntax
             Endpoint:
               Fn::GetAtt: [<QueueName>, Arn]
             Protocol: sqs
+          -
```
Comment (AshwaryaSmridhi) [inline workshop-content/03-project-work/03.4-sns-and-sqs-setup/03.4-sns-and-sqs-setup.md:85]: Anyone reviewing this if you can `serverless deploy` https://github.com/AshwaryaSmridhi/simple-lambda-alert-notification repo that would be really great. Any error triggered should send an SMS but my AWS account is unable to "verify" phone numbers. So I haven't seen any SMS come through.

File: workshop-content/03-project-work/03.4-sns-and-sqs-setup/03.4-sns-and-sqs-setup.md [hunk 2]
```diff
@@ -64,6 +82,9 @@ resources: # CloudFormation template syntax
...
+            Endpoint:'+6146443956784'  # Replace with your phone number
```
Comment (AshwaryaSmridhi) [inline workshop-content/03-project-work/03.4-sns-and-sqs-setup/03.4-sns-and-sqs-setup.md:88]: Anyone reviewing this if you can serverless deploy https://github.com/AshwaryaSmridhi/simple-lambda-alert-notification repo that would be really great. Any error triggered should send an SMS but my AWS account is unable to "verify" phone numbers. So I haven't seen any SMS come through.

Comment (francalovescakes) [reply workshop-content/03-project-work/03.4-sns-and-sqs-setup/03.4-sns-and-sqs-setup.md:88]: is that a real number?

Comment (AshwaryaSmridhi) [reply workshop-content/03-project-work/03.4-sns-and-sqs-setup/03.4-sns-and-sqs-setup.md:88]: No it is not.

**Pull request merged**